Packs/PrismaSaasSecurity/Integrations/SaasSecurityEventCollector/SaasSecurityEventCollector_description.md

```diff
@@ -4,7 +4,8 @@
 2) The max-fetch/limit parameters to fetch events must be divisible by 100.
 3) **reset last fetch** has no effect.
 4) On initial activation this integration will pull events starting from one hour prior.
-5) Calling ```fetch-events``` or ```saas-security-get-events``` or ```test-module``` may take upwards of twenty seconds in some cases.
+5) Using the ```saas-security-get-events``` command may take upwards of twenty seconds in some cases.
+6) In some rare cases more than ```max_fetch``` events could be fetched.
 
 
 Creating the Client ID and Client Secret in the SaaS Security UI:
```

Comment:
Saas

Reply:
i think they only meant to the display on the integration in the yml, no need to change it i guess